packages/tupaia-web/src/layout/TopBar/TopBar.tsx

```diff
@@ -21,8 +21,7 @@ const Header = styled.header<{
   $primaryColor?: string | null;
   $secondaryColor?: string | null;
 }>`
-  background-color: ${({ $primaryColor, theme }) =>
-    $primaryColor || theme.palette.background.default};
+  background-color: #2e2f33;
```

Comment:
Here we do want to use `$primaryColor` if set, because it might come from custom landing pages. We can also use a theme variable for the background color if `$primaryColor` is not set. You could potentially change `theme.palette.background.default` to be this black, since it gets used a lot